.github/workflows/ci-pio.yaml

```
@@ -2,23 +2,6 @@ name: PlatformIO CI
 
 on: [push]
 
```

actionlint (1.7.10)

could not parse as YAML: did not find expected key

(syntax-check)